package.json

```diff
@@ -29,7 +29,7 @@
     "release.docs": "sl-scripts release:docs",
     "release.dryRun": "sl-scripts release --dry-run --debug",
     "storybook": "start-storybook -p 9001",
-    "test": "jest",
+    "test": "jest --silent",
```

Comment:
Is there a way to just silent stdout? We saw spectral silencing stderr which was an actual problem we should have fixed. This was only caught thanks to test harness.

Reply:
I actually want to silent `stderr` here, since I throw errors in most of components, so the output is quite polluted.